### PR TITLE
aredn: hAP ac lite enable 5GHz LAN Access Point

### LINKED ISSUE #215: hAP lite 5Ghz enable dual band capability

### PR DESCRIPTION
**Pull request by ae6xe**
enable ability to bridge LAN physical ports with wireless
LAN Access Point capability. On dual band hAP ac lite,
used in parallel with mesh RF on 2GHz.  Enables future
features on single band devices to turn off mesh RF and
repurposed for mesh LAN Access Point.

closes: #215

### CODE REVIEW

**Comment by ae6xe** [comment]
I've tested this on the following devices:
-hap ac lite
- NS M2 XM
- NS M5 XW
- cpe510v1.1

File: files/www/help.html [hunk 1]
```diff
@@ -407,11 +405,10 @@
 covered here.
 </p>
 <p>
-The other option in the WAN box is the <strong>Mesh Gateway</strong>.
-This is an advanced configuration option.<br>
-When a node has internet access from either the WAN or LAN, that access is
-available to the node itself and to any computer connected to the LAN port.
-When the Mesh Gateway is enabled this node will gate(route) traffic from the
+The <strong>Allow others to use my WAN</strong> is an Advance WAN option.
```
Comment (dman776) [inline files/www/help.html]: change Advance to Advanced

**Review by dman776 — CHANGES_REQUESTED** [review]

File: files/www/help.html (hunk 1 above)
Comment (ae6xe) [reply files/www/help.html]: Corrected.

**Review by dman776 — APPROVED** [review]
tested on my hapaclite and rocket m2

**Comment by AC0WN** [comment]
Build:  aredn-491-a62fe95-mikrotik-rb-nor-flash-16M-ac-sysupgrade.bin

If your SSID has one or more SPACES in the name the 5 GHz radio does not broadcast when the 5G settings are saved and the router rebooted.  We checked this with a WiFi scanner on several different channels.

As soon as you replace the SPACES in the SSID with UNDERLINES <_> the radio works great.

**Comment by ae6xe** [comment]
good catch.   Spaces are allowed.  Will submit a change shortly to allow the spaces to correctly save and pass though the config settings.

**Comment by ae6xe** [comment]
See submitted change:   https://github.com/aredn/aredn_ar71xx/pull/293

**Pull request merged**